patches/chrome-browser-ui-browser.cc.patch

```diff
@@ -1,7 +1,16 @@
 diff --git a/chrome/browser/ui/browser.cc b/chrome/browser/ui/browser.cc
-index 2c9d816e1aff340615061e888cbbcd69300befcd..76f9bc4537210c08f0d696f6f41ec7b9a08a943e 100644
+index 2c9d816e1aff340615061e888cbbcd69300befcd..ff9c60969b10b9037097ea0186cc35c96f0a5a33 100644
 --- a/chrome/browser/ui/browser.cc
 +++ b/chrome/browser/ui/browser.cc
+@@ -434,7 +434,7 @@ Browser::BrowserCreationStatus Browser::GetBrowserCreationStatusForProfile(
+ Browser* Browser::Create(const CreateParams& params) {
+   CHECK_EQ(BrowserCreationStatus::kOk,
+            GetBrowserCreationStatusForProfile(params.profile));
+-  return new Browser(params);
++  return new BraveBrowser(params);
```

Comment:
Maybe #define this as BRAVE_BROWSER_CREATE in the `chromium_src` override and insert it before the chromium's `return` statement?

Reply:
Fixed.